swagger_codegen.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+CURRENT_DIR=$(pwd)
+
+# Defining a temporary directory for cloning
+TMP_DIR=$(mktemp -d)
+
+# Function to clean up the temporary directory
+cleanup() {
+    echo "Cleaning up..."
+    rm -rf "$TMP_DIR"
+}
+
+# Trap to clean up in case of script exit or interruption
+trap cleanup EXIT
+
+curl -o $TMP_DIR/swagger.json https://raw.githubusercontent.com/dydxprotocol/v4-chain/ce926099af057e38d7d79453d01c5ebe0cdc3634/indexer/services/comlink/public/swagger.json
```

Comment:
probably want to update this url to use `main`:
https://raw.githubusercontent.com/dydxprotocol/v4-chain/main/indexer/services/comlink/public/swagger.json